src/lib/utils/testingUtils.ts

```diff
@@ -0,0 +1,4 @@
+export const expectArrayEquivalence = <T>(actual: T[], expected: T[]) => {
+	expect(actual).toEqual(expect.arrayContaining(expected));
+	expect(expected).toEqual(expect.arrayContaining(actual));
+};
```

Comment:
what is the utility of doing this check twice? Does this not care about ordering, essentially?

Reply:
You are correct and yeah thats a better name

Reply:
done